test/parallel/test-stream-readable-resumeScheduled.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+const common = require('../common');
+
+// Testing Readable Stream resumeScheduled state
+
+const assert = require('assert');
+const { Readable, Writable } = require('stream');
+
+{
+  // First test, expect the correct behavior
```

Comment:
I would drop the "First test," "Second test," etc. from the comments. They will likely become out of date as people change the file.